.github/workflows/database.yml

```
@@ -8,15 +8,29 @@ on:
+        run: npm install @prisma/cli
+
+      - name: Migrate Database
+        run: npx prisma deploy
+
   build:
     name: Generate and Build ERD
     runs-on: ubuntu-latest
 
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)